conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conan import ConanFile
+from conan.tools.env import VirtualRunEnv
+from conan.tools.cmake import CMake, CMakeToolchain
+
+required_conan_version = ">=2.1.0"
```

Comment:
Can we lower the Conan version requirement to 2.0? I'd like to use Nix to install Conan, but they only package 2.0.17, which doesn't satisfy the 2.1 requirement. :/
Last time I tried, pip messed up my path with broken CMake binaries.

Reply:
Sorry to hear that pip messed up your path with broken CMake binaries. Yeah of course we can look into lowering the conan version to 2.0. I will also try out Nix too.